api/config/config.go

```diff
@@ -209,18 +209,19 @@ type ClusterConfig struct {
 }
 
 type ImageBuilderConfig struct {
-	ClusterName            string `validate:"required"`
-	GcpProject             string
-	ArtifactServiceType    string
```

Comment:
I removed the field `ArtifactServiceType` here because it's essentially the same as `MlflowConfig.ArtifactServiceType`.